kernel/src/kernel.rs

```diff
@@ -752,41 +803,57 @@ impl Kernel {
                 }
                 process.set_syscall_return_value(rval);
             }
-            Syscall::Yield { which, address } => {
+            Syscall::Yield {
+                which,
+                param1,
+                param2,
```

Comment:
```suggestion
                paramA,
                paramB,
```

Not that it really matters, but might be a bit nicer for readability to match the names used in the TRD